BuildSelectScript.sh

```diff
@@ -347,10 +347,35 @@ while true; do
 
     if [ "$WHICH" = "Loop" ]; then
 
-        # Issue Warning if not done previously
-        open_source_warning
+        section_separator
 
-        run_script "BuildLoopReleased.sh" $CUSTOM_BRANCH
+        echo -e "Until the current dev branch is released as 3.4, this script"
+        echo -e "  has options for the released code (3.2.3) or dev (3.3.0)"
+        echo -e
+        echo -e "      For info on main: see https://github.com/LoopKit/Loop/releases"
+        echo -e "      For info on dev: see https://loopkit.github.io/loopdocs/version/development/#updates-in-dev"
+        echo -e
+        echo -e "If you choose dev, please Choose dev lightly tested"
```

Comment:
Should `Choose` be changed to `choose`?

Reply:
Also maybe users will be confused at what is meant by `dev lightly tested` since they won't see that option until later in the script, so maybe `please choose "dev lightly tested" later in the script` or something?

Reply:
Probably. I copied the menu item string from LoopBuildDev.sh. I will look again. 

Thanks

Reply:
I modified the introductory words and changed some of the order for when information is presented.